src/main/resources/quilt.mod.json

```diff
@@ -5,15 +5,15 @@
 		"id": "tablesaw",
 		"version": "${version}",
 		"metadata": {
-			"name": "Mod Name",
-			"description": "A short description of your mod.",
+			"name": "TableSaw",
+			"description": "A mod that adds a sawmill for cutting wooden blocks.",
 			"contributors": {
-				"Your name here": "Owner"
+				"woodiertexas": "Owner"
```

Comment:
I think this would be better:

"Debuggy Team": "Owner",
"Falkreon": "Contributor; Came up with the idea for Tablesaw"